netlogo-gui/src/main/app/Tabs.scala

```diff
@@ -36,20 +42,11 @@ class Tabs(val workspace:       GUIWorkspace,
   with AfterLoadEvent.Handler
   with ExternalFileSavedEvent.Handler {
 
-  locally {
-    setOpaque(false)
-    setFocusable(false)
-    addChangeListener(this)
-    if (System.getProperty("os.name").startsWith("Mac")) {
-      try {
-        val ui = Class.forName("org.nlogo.app.MacTabbedPaneUI").newInstance.asInstanceOf[ComponentUI]
-        setUI(ui)
-      } catch {
-        case e: ClassNotFoundException =>
-      }
-    }
-  }
+  addChangeListener(this)
 
+  def getTabs = { this }
+
+  def asAbstractTabsPanel = { this.asInstanceOf[AbstractTabsPanel] }
```

Comment:
I don't understand the use for this.  Isn't `Tabs` already an `AbstractTabsPanel`?  Why do we need to cast it?  Why do we need a method to cast it if we do?